server/src/main/java/com/networknt/server/Server.java

```diff
@@ -116,6 +124,10 @@ public static void init() {
 
             // this will make sure that all log statement will have serviceId
             MDC.put(SID, config.getServiceId());
+            
```

Comment:
correlation id may be not needed when the server starts, since correlation id is specific to the requests.